contracts/SportMarkets/SportPositions/SportPositionalMarketManager.sol

```diff
@@ -460,6 +543,27 @@ contract SportPositionalMarketManager is Initializable, ProxyOwned, ProxyPausabl
         }
     }
 
+    function cancelMarketsForParents(address[] memory _parents) external {
+        for (uint i = 0; i < _parents.length; i++) {
+            require(!isDoubleChance[_parents[i]], "Not supported for double chance markets");
```

Comment:
Another approach would be skipping the markets that can not be cancelled, rather than reverting the whole transaction if a single market fails.

Reply:
agree, done.